cypress/e2e/login.cy.js

```diff
@@ -4,27 +4,15 @@ describe("Authentication", () => {
     cy.clearLocalStorage();
   });
 
-  it("will login", () => {
+  it("should allow a valid user to log in", () => {
     cy.visit("/");
-    cy.wait(1000);
-    cy.get(".btn-close:visible").click();
     cy.wait(500);
-    cy.get("button[data-auth='login']:visible").click();
-    cy.wait(1500);
-    cy.get("input[type='email']:visible")
-      .should("exist")
-      .type("bulba@noroff.no");
-    cy.get("input[type='password']:visible").should("exist").type("goldsilver");
-    cy.get(".btn-success:visible").click();
-    cy.window({ log: false }).then((win) => {
-      cy.wrap(null, { timeout: 3000 }).should(() => {
-        const profile = win.localStorage.getItem("profile");
-        const token = win.localStorage.getItem("token");
-        expect(profile).to.not.be.null;
-        expect(token).to.not.be.null;
-      });
-    });
-
-    cy.url().should("include", "profile");
+    cy.get("#registerModal").contains("Login").click();
+    cy.wait(500);
+    cy.get("#loginForm").should("be.visible");
+    cy.get("#loginEmail").type("bulba@noroff.no");
+    cy.get("#loginPassword").type("goldsilver");
+    cy.get("button[type=submit]").contains("Login").click();
```

Comment:
How do you test whether login worked or not? You could look at localStorage

Reply:
Not realy sure what you mean here? I changed this to be more like the invalid-login form with consts rather then typing?

Reply:
I was referring to the localStorage code below, checking the token in local is the best way to check that login worked